script/c91740879.lua

```diff
@@ -63,6 +64,7 @@ end
 function s.rcvtg(e,tp,eg,ep,ev,re,r,rp,chk)
 	local c=e:GetHandler()
 	if chk==0 then return c:GetCounter(COUNTER_CAULDRON)>0 end
+	Duel.Hint(HINT_OPSELECTED,1-tp,e:GetDescription())
```

Comment:
Cauldron of the Old Man: Announce which effect has been chosen
While not strictly necessary, for stylistic consistency with e.g. Zoodiac monsters this could, and arguably should, be implemented as a cost function that both effects can share.